Sources/MapboxDirections/RouteStep.swift

```diff
@@ -319,17 +321,7 @@ struct Road {
     }
     
     init(name: String, ref: String?, exits: String?, destination: String?, rotaryName: String?) {
-        if !name.isEmpty, let ref = ref {
-            // Mapbox Directions API v5 encodes the ref separately from the name but redundantly includes the ref in the name for backwards compatibility. Remove the ref from the name.
```

Comment:
Directions API v5 profiles powered by Valhalla no longer include the ref in the name. However, the mapbox/cycling profile, which is powered by OSRM, [still includes the ref](https://api.mapbox.com/directions/v5/mapbox/cycling/-84.49966248187295,39.10237033604159;-84.49950427649345,39.101791247428054.json?steps=true&overview=full&geometries=geojson&roundabout_exits=true&access_token=).

/ref Project-OSRM/osrm-backend#2704